lib/insights/api/common/application_controller_mixins/api_doc.rb

```diff
@@ -21,7 +21,7 @@ module ClassMethods
             private
 
             def api_doc
-              @api_doc ||= ::ManageIQ::API::Common::OpenApi::Docs.instance[api_version[1..-1].sub(/x/, ".")]
```

Comment:
There was a namespace issue here, that's why this is fully qualified

Reply:
I'm going to back out all of the namespace changes...can always do that separately